packages/edit-post/src/components/header/template-title/style.scss

```diff
@@ -13,5 +13,22 @@
 }
 
 .edit-post-template-top-area__popover .components-popover__content {
-	min-width: 360px;
+	min-width: 280px;
+}
+
+.edit-post-template-top-area__second-menu-group {
+	margin-left: - $grid-unit-15;
```

Comment:
Oh interesting with the space after the minus! I'd usually do -$grid-unit-15 for negative numbers. Not really something that needs fixing, just stood out.

Reply:
I've had issues with not putting a space in the past, so put one in habitually now. I'll check if it's actually needed though.

Reply:
I assume it's fine, just looking at the code here while I'm rebooting my environment testing your other PR.